examples/OrmliteExample/.idea/compiler.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The whole .idea folder and iml files are not needed. They are redundant, and generated from the gradle project descriptor which contains all information.